modules/cpuinfo/0.0.0-20250925-877328f/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
```

Comment:
Add - ubuntu 24.04 (and maybe ubunutu 22.04) - you can drop ubuntu 2004 (or keep it as you wisch)